Source/Core/Core/BootManager.cpp

```diff
@@ -58,8 +59,138 @@ struct ConfigCache
 	TEXIDevices m_EXIDevice[MAX_EXI_CHANNELS];
 	std::string strBackend, sBackend;
 	std::string m_strGPUDeterminismMode;
+
+public:
```

Comment:
They are, but IMO [this](https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/InputCommon/ControllerInterface/evdev/evdev.h#L20-L83) is pretty terrible, even though it makes sense why they did it that way.